services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+import * as _ from 'lodash';
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  // model: juggler.ModelBase | any;
+  model: juggler.PersistedModel | any;
```

Comment:
I do intend to ditch my own repository implementation here; it mostly came about as a way of debugging the juggler import issues that I was running into previously. Now that it's not an issue, I can switch the controller to use a `DefaultCrudRepository` instead (which I'm working on now).